minet/cli/youtube/videos.py

```diff
@@ -27,11 +117,48 @@ def videos_action(namespace, output_file):
         unit=' videos',
     )
 
-    for line in enricher:
+    for chunk in gen_chunks(enricher):
+
+        all_ids = []
+        for i in chunk:
+            if i[0]:
+                all_ids.append(i[0])
+
+        list_id = ",".join(all_ids)
+
+        url = URL_TEMPLATE % {'list_id': list_id, 'key': namespace.key}
+        http = create_pool()
+        err, response, result = request_json(http, url)
+
+        if err:
+            die(err)
+        elif response.status == 403:
+            time.sleep(wait())
+        elif response.status >= 400:
+            die(response.status)
+
+        data = get_data(result)
+
+        id_available = []
+        for sub_list in data:
+            id_available.append(sub_list[0])
+
+        not_available = list(set(all_ids).difference(set(id_available)))
```

Comment:
Pourquoi ne pas garder `not_available` comme un set et le convertir en liste sachant que tu fais des `in` dedans plus tard qui seraient plus rapides avec un set?